weather_app/management/commands/load_files_data.py

```diff
@@ -0,0 +1,23 @@
+from django.core.management.base import BaseCommand
+import os
+import csv
+from .utils import reading_files,data_sending_to_database
+
+""" Base Command class  """
+class Command(BaseCommand):
+    help = "Loading data from files"
+    """ 
+    command function handling here actions
+    1: reading_files()
+    2: data_sending_to_database()
+    both function are using in it
+    """
+    def handle(self, *args, **options):
+        path_root = os.path.abspath(os.path.dirname(__name__))
```

Comment:
why using the abspath, we can define path from the user how runs the program, see expanduser method of os library